tensorbay/opendataset/_utility/voc.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""_get_data function for VOC2012Detection dataset and UrbanObjectionDetection dataset."""
```

Comment:
this module string not good, because if `_get_data` is used in a new loader, this docstring also need to be updated.